libs/sdk-core/src/breez_services.rs

```diff
@@ -905,6 +909,38 @@ impl BreezServices {
         Ok(())
     }
 
+    fn persist_new_bolt11_payment_attempt(
+        &self,
+        invoice: &LNInvoice,
+        amount_msat: u64,
+    ) -> Result<(), SendPaymentError> {
+        self.persister.insert_or_update_payments(&[Payment {
+            id: invoice.payment_hash.clone(),
+            payment_type: PaymentType::Sent,
+            payment_time: SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs() as i64,
+            amount_msat,
+            fee_msat: 0,
+            status: PaymentStatus::Pending,
+            description: invoice.description.clone(),
+            details: PaymentDetails::Ln {
+                data: LnPaymentDetails {
+                    payment_hash: invoice.payment_hash.clone(),
+                    label: String::new(),
+                    destination_pubkey: invoice.payee_pubkey.clone(),
+                    payment_preimage: String::new(),
+                    keysend: false,
+                    bolt11: invoice.bolt11.clone(),
+                    lnurl_success_action: None,
+                    ln_address: None,
+                    lnurl_metadata: None,
+                    lnurl_withdraw_endpoint: None,
+                    swap_info: None,
+                },
+            },
+        }])?;
+        Ok(())
```

Comment:
```suggestion
        self.persister.insert_payment_external_info(
            &invoice.payment_hash,
            None,
            None,
            None,
            None,
            amount_msat,
        )?;
        Ok(())
```

Reply:
Addressed in [49e5938](https://github.com/breez/breez-sdk/pull/623/commits/49e5938166a2793814d74b33000e387e93a43e9a)